components/hacbs/integration-service/base/kustomization.yaml

```diff
@@ -1,12 +1,12 @@
 resources:
-- "https://github.com/redhat-appstudio/integration-service/config/kcp?ref=cf8459fbde3cfa5f92518e637fb55150500b8626"
+- "https://github.com/redhat-appstudio/integration-service/config/kcp?ref=ffcbd023df312f6d318543ac5b2bc69230666cc1
```

Comment:
@hongweiliu17 here is a bug, at the end the `"` is removed, usual fix for this is to remove quotes

Reply:
I don't have permission to push code to https://github.com/redhat-appstudio/infra-deployments/tree/integration-service

Reply:
pushed and merged

Reply:
Thank you!